frontend/.prettierrc.json

```diff
@@ -0,0 +1,6 @@
+{
+  "trailingComma": "es5",
+  "tabWidth": 2,
+  "semi": true,
+  "singleQuote": false
```

Comment:
Consider including an `"endOfLine"` option for consistency across operating systems.